### ISSUE TITLE
55530804 - Dark Messiah of Might & Magic Elements

### ISSUE THREAD

**Issue by kanelakis** [xenia-project/game-compatibility#1860]
[Marketplace]
https://marketplace.xbox.com/it-IT/Product/DMMM-Elements/66acd000-77fe-1000-9115-d80255530804

Tested on https://github.com/xenia-project/xenia/commit/canary_experimental@a0e27f3a0

# Issues:
In Game
# Log:
[xenia.zip](https://github.com/xenia-project/game-compatibility/files/7668252/xenia.zip)

# Screenshot(s):
[URL=https://www.imagebam.com/view/ME5GZ2S][IMG]https://thumbs4.imagebam.com/b9/8b/d1/ME5GZ2S_t.png[/IMG][/URL] [URL=https://www.imagebam.com/view/ME5GZ2T][IMG]https://thumbs4.imagebam.com/c4/ec/39/ME5GZ2T_t.png[/IMG][/URL] [URL=https://www.imagebam.com/view/ME5GZ2U][IMG]https://thumbs4.imagebam.com/5c/70/d2/ME5GZ2U_t.png[/IMG][/URL] 

# Labels:
https://github.com/xenia-project/game-compatibility/labels/gpu-shader-errors

**Comment by kanelakis** [comment]
If there is something to correct let me know.

**Comment by Gliniak** [comment]
Yes there is. It is compatibility report list of official Xenia builds not canary builds. Thank you.

**Comment by Etokapa** [comment]
Also known as **DMMM:Elements**

Tested on https://github.com/xenia-project/xenia/commit/b64b4c676

Started a new game and kicked 8 men to death. Xenia froze while exploring. 15-25FPS with vsync off and shaders cached.

:warning:The game will be too bright unless you set `d3d12_readback_resolve=true`. This will severely impact performance though.
:warning:Low FPS despite low CPU and GPU usage.
:mute:Audio starts and then stops.
:high_brightness:Lights are not occluded.
:grey_question:Shadows from enemies render as squares?
:grey_question:Xenia will often just freeze during intros?

![dar (9)](https://user-images.githubusercontent.com/31154606/147834063-1e465483-05ff-4e9f-86fb-4baf70085efb.png)

<details>
  <summary>More Screenshots</summary>

![dar (4)](https://user-images.githubusercontent.com/31154606/147834075-97ee5dcd-d2de-4ab3-ae6e-f66748ffad77.png)
![dar (5)](https://user-images.githubusercontent.com/31154606/147834076-67355ad9-b581-49e6-afc3-7f20bbaaa920.png)
![dar (6)](https://user-images.githubusercontent.com/31154606/147834077-89e41b1b-596f-4b36-b44d-9f17f7c9e86c.png)
![dar (7)](https://user-images.githubusercontent.com/31154606/147834078-bbe40880-ce40-4a89-be21-078812234cb4.png)
![dar (8)](https://user-images.githubusercontent.com/31154606/147834079-e7e1622a-670f-49ca-83eb-b411727ac558.png)
![dar (10)](https://user-images.githubusercontent.com/31154606/147834080-bfb6eb19-fff0-40bb-a640-2293ed22b4a5.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/7796758/xenia.zip)
[xenia.z01.zip](https://github.com/xenia-project/game-compatibility/files/7796757/xenia.z01.zip)

state-gameplay | tech-engine-source | tech-middleware-havok | gpu-readback

**Comment by Margen67** [comment]
Xenia Canary issue: https://github.com/xenia-canary/game-compatibility/issues/222
